forloop_modules/utils/various.py

```diff
@@ -48,7 +49,9 @@ def serialize_dataframe_to_api(variable_value_df: pd.DataFrame) -> dict:
     :return: Variable's value serialized as a dict
     :rtype: dict
     """
-    return {"columns": list(variable_value_df.columns), "values": variable_value_df.values.tolist()}
+    df = variable_value_df.copy()
+    df = df.replace(np.nan, None)
```

Comment:
first call tolist(), then replace elements to None -> sounds it would be better to me